src/helpers/serviceSelector.ts

```diff
@@ -70,16 +94,23 @@ export default class ServiceSelector {
     // Implement source event based descision logic for selecting service
   }
 
-  public static getDestinationServiceByName(name: string): DestinationService {
-    switch (name) {
-      case INTEGRATION_SERVICE.CDK_V1_DEST:
-        return this.fetchCachedService(INTEGRATION_SERVICE.CDK_V1_DEST);
-      case INTEGRATION_SERVICE.CDK_V2_DEST:
-        return this.fetchCachedService(INTEGRATION_SERVICE.CDK_V2_DEST);
-      case INTEGRATION_SERVICE.NATIVE_DEST:
-        return this.fetchCachedService(INTEGRATION_SERVICE.NATIVE_DEST);
-      default:
-        throw new PlatformError('Invalid Service');
+  public static getDestinationService(
```

Comment:
document example destination definition config format for this

Reply:
You mean adding new type for config?

Reply:
Yes, how the destination config should be created for this feature